pkg/rpcclient/nns/contract.go

```diff
@@ -44,6 +69,21 @@ func NewReader(invoker Invoker, hash util.Uint160) *ContractReader {
 	return &ContractReader{*nep11.NewNonDivisibleReader(invoker, hash), invoker, hash}
 }
 
+// Hash contains contract hash.
+var Hash = util.Uint160{0xde, 0x46, 0x5f, 0x5d, 0x50, 0x57, 0xcf, 0x33, 0x28, 0x47, 0x94, 0xc5, 0xcf, 0xc2, 0xc, 0x69, 0x37, 0x1c, 0xac, 0x50}
```

Comment:
We need a dynamic hash. Run `generate-rpcwrapper` without `--hash` flag and merge s the result with an existing code. There shouldn't be `Hash` field.